identity/src/main/java/com/android/identity/credentialtype/CredentialAttributeType.kt

```diff
@@ -26,13 +26,13 @@ package com.android.identity.credentialtype
  * for inputting or displaying credentials attributes.
  */
 sealed class CredentialAttributeType {
-    object STRING : CredentialAttributeType()
-    object NUMBER : CredentialAttributeType()
-    object DATE : CredentialAttributeType()
-    object DATE_TIME : CredentialAttributeType()
-    object PICTURE : CredentialAttributeType()
-    object BOOLEAN : CredentialAttributeType()
-    object COMPLEX_TYPE : CredentialAttributeType()
+    object String : CredentialAttributeType()
+    object Number : CredentialAttributeType()
+    object Date : CredentialAttributeType()
+    object DateTime : CredentialAttributeType()
+    object Picture : CredentialAttributeType()
+    object Boolean : CredentialAttributeType()
+    object ComplexType : CredentialAttributeType()
```

Comment:
Why are you changing this to camel-case? Seems clearly in violation of the [guidelines](https://kotlinlang.org/docs/coding-conventions.html#property-names), which states _"... or top-level or object val properties with no custom get function that hold deeply immutable data ... "_

Reply:
Sorry for a drive-by comment, but I think you're misinterpreting the guidelines. This isn't _top level_ or _object val property_, it's an [object](https://kotlinlang.org/docs/coding-conventions.html#naming-rules) instead:

> Names of classes and **objects** start with an uppercase letter and use the camel case:

(Emphasis mine)

Reply:
@davidz25 totally agree for `const val` - the convention is to upper case them, however, from the link you posted:

"_Names of properties holding references to singleton objects can use the same naming style as object declarations:_"
```
val PersonComparator: Comparator<Person> = /*...*/
```

Reply:
OK, no worries, TIL!